stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/MessageComposer.kt

```diff
@@ -292,7 +292,7 @@ public fun MessageComposer(
             mentionPopupContent(mentionSuggestions)
         }
 
-        if (commandSuggestions.isNotEmpty()) {
+        if (commandSuggestions.isNotEmpty() && messageComposerState.attachments.isEmpty()) {
```

Comment:
Maybe add attachments to destructive declaration above?
`    val (_, attachments, activeAction, validationErrors, mentionSuggestions, commandSuggestions) = messageComposerState`